app/endpoints.py

```diff
@@ -1199,6 +1199,113 @@ def find_subject_dx_implications(
+        query["testDate"] = {}
+
+        for date_range in testDateRange:
+            query["testDate"][date_range['OPERATOR']] = date_range['DATE']
+
+    # Subject Query
+    query["patientID"] = {"$eq": subject}
+
+    # testIdentifiers query
+    if testIdentifiers:
+        testIdentifiers = list(map(str.strip, testIdentifiers))
+        query["testID"] = {"$in": testIdentifiers}
+
+    # specimenIdentifiers query
+    if specimenIdentifiers:
+        specimenIdentifiers = list(map(str.strip, specimenIdentifiers))
+        query["specimenID"] = {"$in": specimenIdentifiers}
+
+    # Genomic Source Class Query
+    if genomicSourceClass:
+        genomicSourceClass = genomicSourceClass.strip().lower()
+        query["genomicSourceClass"] = {"$eq": genomicSourceClass}
+
+    normalized_variants = []
+    if ranges:
+        ranges = list(map(common.get_range, ranges))
+        common.get_lift_over_range(ranges)
+        variants = common.get_variants(ranges, query)
+        if not variants:
+            return jsonify({"resourceType": "Parameters"})
+        normalized_variants = [{variant["BUILD"]: variant["SPDI"]} for variant in variants]
+
+    if variants and not ranges:
+        normalized_variants = list(map(common.get_variant, variants))
+
+    # Result Object
+    result = OrderedDict()
+    result["resourceType"] = "Parameters"
+    result["parameter"] = []
+
+    if normalized_variants:
+        if not ranges:
+            genomics_build_presence = common.get_genomics_build_presence(query)
+
+            for normalizedVariant in normalized_variants:
+                if not normalizedVariant["GRCh37"] and genomics_build_presence["GRCh37"]:
+                    abort(422, f'Failed LiftOver. Variant: {normalizedVariant["variant"]}')
+                elif not normalizedVariant["GRCh38"] and genomics_build_presence["GRCh38"]:
+                    abort(422, f'Failed LiftOver. Variant: {normalizedVariant["variant"]}')
+
+        query_results = common.query_molecular_consequences_by_variants(normalized_variants, normalized_feature_consequence_list, query)
+
+        for res in query_results:
+            if res["molecularConsequenceMatches"]:
+                # ref_seq = common.get_ref_seq_by_chrom_and_build(res['genomicBuild'], res['CHROM'])
+                for molecular_consequence in res["molecularConsequenceMatches"]:
+                    parameter = OrderedDict()
+                    parameter["name"] = "Molecular Consequence"
+                    parameter["part"] = []
+
+                    molecular_consequence_profile = common.create_molecular_consequence_profile(molecular_consequence, subject, [str(res['variantID'])])
+                    parameter["part"].append({
+                        "name": "Molecular Consequence",
+                        "resource": molecular_consequence_profile
+                    })
+                    # resource = common.create_fhir_variant_resource(
```

Comment:
Please remove commented-out code or add some comment to explain why it exists (same for line 1280)